envoimoinscher.php

```diff
@@ -2811,7 +3041,7 @@ public function makeApiCall($params)
 		$offers_quotation = array();
 		if (count($offers_db) == 1) // correction needed for quotation
 		{
-			$cot_cl->getQuotation($quot_info);
+			@$cot_cl->getQuotation($quot_info);
```

Comment:
Surrounding with a try/catch should be less dirty than a ```@``` and will allow you to manage an error properly.